app/javascript/utils/form/final_form/Field.js

```diff
@@ -106,6 +107,7 @@ export const SelectField = ({ fieldName, label, blockNote, validation, id, optio
               id={id || `form-${fieldName}`}
               name={input.name}
               className={classNames(className, (meta.error && meta.touched && 'error') || '')}
+              {...(selectValue && { value: selectValue })}
```

Comment:
did we not have the option before of having a default value for a select field?

Reply:
I did not see this in place, at least in the way React recommends so I added this